x-pack/plugins/osquery/common/search_strategy/osquery/index.ts

```diff
@@ -34,8 +34,15 @@ export enum OsqueryQueries {
 
 export type FactoryQueryTypes = OsqueryQueries;
 
+export type AggregationValue = string | { field: string; subaggs: Aggregation };
```

Comment:
i ended up using the `AggregationContainer` type, that way we can just pass in whatever aggregations we want. it might be a little unwieldy, but at least we are tied into the canonical types. let me know what you think!